homeassistant/components/wallbox/strings.json

```
@@ -114,6 +114,12 @@
       }
     }
   },
+  "issues": {
+    "insufficient_rights": {
+      "title": "The Wallbox account has insufficient rights.",
+      "description": "The Wallbox account has insufficient right to lock/unlock and change the charging power. Please assign the user admin rights in the Wallbox portal."
```

We should use "rights" in both parts of the sentence and match the title:

```suggestion
      "description": "The Wallbox account has insufficient rights to lock/unlock and change the charging power. Please assign the user admin rights in the Wallbox portal."
```